ports/servo/glutin_app/window.rs

```diff
@@ -483,6 +485,14 @@ impl Window {
                     self.event_queue.borrow_mut().push(WindowEvent::Resize);
                 }
             }
+            Event::Suspended(suspended) => {
+                self.suspended.set(suspended);
+                if suspended {
+                    self.set_animation_state(AnimationState::Idle);
```

Comment:
Animation state is correctly set to Idle here, but it seems that it's overwritten by the compositor window.set_animation_state() call of the current tick. This starts the event loop again and wastes battery when the Android app is in background.

Reply:
@MortimerGoro can you try by adding a check to `suspended` in `fn is_animating()`. Return false if not animating or if suspended? Would that be enough?

Reply:
It works :)

```
fn is_animating(&self) -> bool {
    self.animation_state.get() == AnimationState::Animating && !self.suspended.get()
}
```